Add Copilot developer agent billing note (#9137)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform/billing.md

A defined number of <Constant name="copilot" /> invocations is allocated monthly based on your [subscription plan](https://www.getdbt.com/pricing). Once the usage limit is reached, access to Copilot functionality will be temporarily disabled until the start of the next billing cycle.

> The Copilot developer agent (Public Preview) currently draws from your available Copilot request allotment. Pricing and usage are subject to change.

Author: ```suggestion
The <Constant name="copilot"/> <Constant name="dev_agent"/>  (Public Preview) currently draws from your available <Constant name="copilot"/> request allotment. Pricing and usage are subject to change.
```

> The <Constant name="copilot"/> <Constant name="dev_agent"/>  (Public Preview) currently draws from your available <Constant name="copilot"/> request allotment. Pricing and usage are subject to change.

Author: ```suggestion
The <Constant name="copilot"/> <Constant name="dev_agent"/>  (Public preview) currently draws from your available <Constant name="copilot"/> request allotment. Pricing and usage are subject to change.
```

> The <Constant name="copilot"/> <Constant name="dev_agent"/>  (Public preview) currently draws from your available <Constant name="copilot"/> request allotment. Pricing and usage are subject to change.

Reviewer: ```suggestion
The <Constant name="copilot"/> <Constant name="dev_agent"/> (public preview) currently draws from your available <Constant name="copilot"/> request allotment. Pricing and usage are subject to change.
```
